condensation.gemspec

```
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'timecop'
   spec.add_development_dependency 'rubocop'
 
-  spec.add_dependency 'liquid', '~> 4.0.3'
+  spec.add_dependency 'liquid', '~> 5.0.1'
```

This would get us that `base64_encoded` functionality
```suggestion
  spec.add_dependency 'liquid', '~> 5.3.0'
```